jobs.go

```diff
@@ -45,6 +45,7 @@ type Job struct {
 	ArtifactsExpireAt *time.Time `json:"artifacts_expire_at"`
 	ID                int        `json:"id"`
 	Name              string     `json:"name"`
+	TagList           []string   `json:"tag_list"`
```

Comment:
Please place this between `ArtifactsExpireAt` and `ID`, following the ordering in the docs.

Reply:
Done!